tests/gravity_quadrature_degree_increase/screen-output

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 2.4.0-pre (changelowerlimit, 55a563b75)
+--     . using deal.II 9.3.1
+--     .       with 32 bit indices and vectorization level 1 (128 bits)
+--     . using Trilinos 12.18.1
+--     . using p4est 2.2.0
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-- For information on how to cite ASPECT, see:
+--   https://aspect.geodynamics.org/citing.html?ver=2.4.0-pre&sha=55a563b75&src=code
+-----------------------------------------------------------------------------
+Number of active cells: 12 (on 1 levels)
+Number of degrees of freedom: 628 (450+28+150)
+
+*** Timestep 0:  t=0 years, dt=0 years
+
+   Postprocessing:
+     Writing gravity output: output-gravity_quadrature_degree_increase/output_gravity/gravity-00000
+
+Termination requested by criterion: end time
+
+
++---------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start    |      1.24s |            |
+|                                             |            |            |
+| Section                         | no. calls |  wall time | % of total |
++---------------------------------+-----------+------------+------------+
+| Initialization                  |         1 |     0.895s |        72% |
+| Postprocessing                  |         1 |   0.00818s |      0.66% |
+| Setup dof systems               |         1 |     0.116s |       9.4% |
+| Setup initial conditions        |         1 |     0.132s |        11% |
+| Setup matrices                  |         1 |    0.0134s |       1.1% |
++---------------------------------+-----------+------------+------------+
+
+-----------------------------------------------------------------------------
+-- For information on how to cite ASPECT, see:
+--   https://aspect.geodynamics.org/citing.html?ver=2.4.0-pre&sha=55a563b75&src=code
+-----------------------------------------------------------------------------
```

Comment:
Could you delete the version/wallclock/blurb lines (basically, any lines which start with `--`, `|` or `+`)?
Alternatively, if you ran your test with ctest, output files with the correct format are stored as 
`tests/output-<name-of-test>/<name-of-file>.notime`